pages/favorites.js

```diff
@@ -1,3 +1,21 @@
-export default function Favorites() {
-  return <h2>favorite page</h2>;
+import FavoriteCard from "@/components/FavoriteCard";
+
+export default function Favorites({
+  isFavorite,
+  favoritePlaces,
+  onToggleFavorite,
+}) {
+  if (favoritePlaces.length === 0) {
+    return <h2>Wow, so much empty...</h2>;
+  } else {
+    return (
+      <>
+        <FavoriteCard
+          isFavorite={isFavorite}
+          favoritePlaces={favoritePlaces}
+          onToggleFavorite={onToggleFavorite}
+        />
+      </>
+    );
+  }
```

Comment:
you could use a ternary here: `!favoritePlaces ? <h2>Wow...</h2> : <FavoriteCard .../>`
also. you don't need the Fragments around the FavoriteCard